prediction_market_agent_tooling/tools/cow/cow_order.py

```
@@ -14,17 +14,18 @@
     OrderQuoteSideKindSell,
     OrderStatus,
     TokenAmount,
 )
 from eth_account.signers.local import LocalAccount
 from eth_typing.evm import ChecksumAddress
 from web3 import Web3
-from web3.types import Wei
 
 from prediction_market_agent_tooling.config import APIKeys
-from prediction_market_agent_tooling.gtypes import ChecksumAddress, Wei, wei_type
+from prediction_market_agent_tooling.gtypes import ChecksumAddress, Wei
```

Ruff (0.8.2)

Redefinition of unused `ChecksumAddress` from line 24

(F811)